src/main.cpp

```diff
@@ -1,18 +1,77 @@
 #include <pros/vision.hpp>
 #include <pros/rtos.hpp>
+#include <cmath>
+#include <optional>
+#include <iostream>
 #include "main.hpp"
 
 void initialize() {}
 void disabled() {}
 void competition_initialize() {}
 void autonomous() {}
 
-#define VISION_PORT 1
+namespace ports {
+	constexpr int VISION = 1;
+}
+
+namespace measures {
+	constexpr int GOAL_HEIGHT = 64; // 25 in to cm
+	constexpr int SENSOR_HEIGHT = 44; // 17.5 in to cm
+	constexpr float VERTICAL_FOV = 0.715585; // 41 deg to radians
+	constexpr float LENS_ANGLE = 0.261799; // 15 deg to radians
+}
+
+constexpr int SIG_ERR = 255;
```

Comment:
use uint32_t to match struct definition

Reply:
As well, move this into config namespace:
```cpp
// main.hpp
namespace config {
    constexpr uint32_t SIG_ERR = 255; // can be accessed as config::SIG_ERR
};
```

Reply:
It's not config